syntax = "proto3";

message Monk {
  string name = 1;
  string message = 2;
}

message UnaryRequest {}

message StreamRequest { int32 delay = 1; }

service HelloMonastery {
  rpc SayHelloMonk(UnaryRequest) returns (Monk);

  rpc SayHelloMonks(StreamRequest) returns (stream Monk);
}